lib/src/match.dart

```diff
@@ -149,10 +149,10 @@ int matchBitap(String text, String pattern, int loc, double threshold,
     int start = max(1, loc - bin_mid + 1);
     int finish = min(loc + bin_mid, text.length) + pattern.length;
 
-    final rd = new List<int>(finish + 2);
+    final rd = new List<int?>.filled(finish + 2, 0);
```

Comment:
i cant see a point where the List gets an null entry. so it could be 
List<int>